src/components/core/PivotTable.tsx

```diff
@@ -1,23 +1,53 @@
 // (C) 2007-2018 GoodData Corporation
 import * as React from 'react';
 import { AgGridReact } from 'ag-grid-react';
+import * as classnames from 'classnames';
```

Comment:
as classNames please, so it is consistent with the codebase

Reply:
Because everyone else including Vojty, Vašek and Petr Beneš think it's best. I actually don't know why. I guess typescript bullshit or tree-shaking

Reply:
done